packages/create-plugin/src/utils/utils.packageManager.ts

```diff
@@ -107,3 +107,18 @@ function getPackageManagerFromPackageJson(): PackageManager | undefined {
 
   return undefined;
 }
+
+export function getPackageManagerExecute() {
+  const packageManager = getPackageManagerWithFallback();
+
+  switch (packageManager.packageManagerName) {
+    case 'yarn':
+      return 'yarn dlx';
```

Comment:
I'm afraid there is no npx equivalent for yarn classic which (to the best of my knowledge) is the only version of yarn create-plugin supports right now.

```
$ yarn dlx prettier@2 . --write
yarn run v1.22.19
error Command "dlx" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

As yarn classic puts packages on the filesystem in a very similar manner to npm you might be able to get away with using the default case (`npx`) for yarn here too.

Reply:
since we are going to always use `npx` I totally removed this function